### DISCUSSION TITLE
Contributing Guidelines

### DISCUSSION THREAD

**Discussion by mkosater** [dbeaver/dbeaver#12403] [DBeaver Development]
I was looking for the contributing guidelines for the project I did not see them in the source repo nor the discussion or wiki (possibly have missed them).  I submitted a couple of issues and I have fixes, but wanted to understand the process before submitting PRs.

**Comment by p91paul** [comment]
As far as I have seen, the process is: you open a PR, they review it, they merge it. 😁

**Comment by LonwoLonwo** [comment]
Hello @mkosater 
p91paul is basically right. Part of instruction about project clone, branch and PR you can find here: https://github.com/dbeaver/dbeaver/wiki/Localization
